lib/modules/accountSwitcher.js

```diff
@@ -142,13 +149,33 @@ modules['accountSwitcher'] = {
 
 							RESUtils.getUserInfo(function(userInfo) {
 								var userDetails = username;
+								var $gildedSpan = $('<span>', {
+									'class': 'gilded-icon',
+								});
+								$gildedSpan.css('display', 'none');
```

Comment:
This seems superfluous since the element doesn't even get added to the DOM until later.